docs/content/app-protect/installation.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Install with NGINX App Protect
+title: Install NGINX Ingress Controller with App Protect
```

Comment:
Is the converter script is adding a newline after the title in the metadata?

Reply:
It shouldn't be doing that but I'll check.

Reply:
Yes.  It is.  I updated the script and re-ran it.